src/cart/crt_rpc.c

```diff
@@ -226,10 +231,16 @@ char
 		CRT_ST_RPCS_LIST
 		CRT_CTL_RPCS_LIST
 	}
-	return "DAOS";
+
+	switch (opc >> CRT_PROTO_BASEOPC_SHIFT) {
```

Comment:
Do you only want to come into this code for some values of the upper bits (basically the protocol id?)

Reply:
Hmm, not sure i understand.
Upper 8 bits are for the opcode, while bottom 16 are or the protocol, so we are extracting base opcode here and comparing it to base daos opcodes.

262 #define CRT_PROTO_BASEOPC_MASK  (0xFF000000UL)
263 #define CRT_PROTO_BASEOPC_SHIFT 24
264 #define CRT_PROTO_VER_MASK      (0x00FF0000UL)
265 #define CRT_PROTO_COUNT_MASK    (0x0000FFFFUL)